src/client/datascience/data-viewing/dataViewerFactory.ts

```diff
@@ -46,4 +60,29 @@ export class DataViewerFactory implements IDataViewerFactory, IAsyncDisposable {
         }
         return result;
     }
+
+    private updateOpenDataViewers(viewer: IDataViewer) {
+        this.knownViewers.delete(viewer);
+    }
+
+    private async updateViewStateContext() {
+        // A data viewer's view state has changed. Look through our known viewers to see if any are active
+        let hasActiveViewer = false;
+        this.knownViewers.forEach((viewer) => {
+            if (viewer.active) {
+                hasActiveViewer = true;
+            }
+        });
+        await this.viewContext.set(hasActiveViewer);
```

Comment:
Didn't think it really made sense to track this context in the activeEditorContext since the data viewer is not an editor and the dataViewerFactory already tracks all active data viewers.